src/test/java/utils/TakeScreens.java

```diff
@@ -0,0 +1,53 @@
+package utils;
+
+import com.cucumber.listener.Reporter;
+import org.openqa.selenium.OutputType;
+import org.openqa.selenium.TakesScreenshot;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.io.FileHandler;
+import ru.yandex.qatools.ashot.AShot;
+import ru.yandex.qatools.ashot.Screenshot;
+import ru.yandex.qatools.ashot.shooting.ShootingStrategies;
+import javax.imageio.ImageIO;
+import java.io.File;
+import java.io.IOException;
+import com.google.common.io.Files;
+import java.text.SimpleDateFormat;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.Date;
+public class TakeScreens {
+    private static DateTimeFormatter dtf = DateTimeFormatter.ofPattern("yyyy_MM_dd_HH_mm_ss");
+    private static LocalDateTime now = LocalDateTime.now();
+    private static String nowTime = dtf.format(now);
+    private static String destFile = System.getProperty("user.dir") + "/target/screenshots";
+
+    public static void takeScreenshot(WebDriver driver, String screenshotName) {
+        String timestamp = new SimpleDateFormat("yyyy_MM_dd__hh_mm_ss").format(new Date());
+        try {
+            File fileSource = ((TakesScreenshot) driver).getScreenshotAs(OutputType.FILE);
+            File destinationPath = new File(System.getProperty("user.dir") + "/target/cucumber-reports/screenshots/" + timestamp + "_" + screenshotName + ".png");
+            //Copy taken screenshot from source location to destination location
+            Files.copy(fileSource, destinationPath);
+            //This attach the specified screenshot to the test
```

Comment:
instead of adding here comments, please add a java doc

Reply:
Done